core/templates/dev/head/css/oppia.css

```diff
@@ -839,6 +867,8 @@ textarea {
 
 .navbar-default .navbar-container {
   background: #009688;
+
```

Comment:
I'm referring to lines 870 and 871. Are those occurring in your local environment?